setup.py

```diff
@@ -20,7 +20,7 @@
 
 setuptools.setup(
     name="python-ambariclient",
-    version="0.6.0",
+    version="0.7.0",
```

Comment:
I don't think a change like this warrants a minor version increment. I'd just call it 0.6.1